chia/full_node/full_node.py

```diff
@@ -164,23 +163,8 @@ async def _start(self):
         # These many respond_transaction tasks can be active at any point in time
         self.respond_transaction_semaphore = asyncio.Semaphore(200)
         # create the store (db) and full node instance
-        self.connection = await aiosqlite.connect(self.db_path)
-        await self.connection.execute("pragma journal_mode=wal")
-        db_sync = db_synchronous_on(self.config.get("db_sync", "auto"), self.db_path)
-        self.log.info(f"opening blockchain DB: synchronous={db_sync}")
-        await self.connection.execute("pragma synchronous={}".format(db_sync))
-
-        async with self.connection.execute(
```

Comment:
this was the part that made all the tests fail. I re-jiggered it a bit to do this after creating the DBWrapper and using it.